mockdata-daemon/src/main/java/org/msse/demo/kafka/KafkaDaemon.java

```diff
@@ -59,11 +60,11 @@ public void createEvent() {
     musicService.createEvent();
   }
 
-  // every 20 seconds
-  @Scheduled(cron = "*/60 * * * * *")
-  public void createVenue() {
-    musicService.createVenue();
-  }
+//  // every 20 seconds
+//  @Scheduled(cron = "*/60 * * * * *")
+//  public void createVenue() {
+//    musicService.createVenue();
+//  }
```

Comment:
I think now that we have a static set of "real" venues we can just clean out this random venue generation. You can remove all of this commented code.